Source/VMware.vSphereDSC/DSCResources/VMHostCache.ps1

```diff
@@ -109,11 +112,19 @@ class VMHostSSDCache : VMHostBaseDSC {
     <#
     .DESCRIPTION
 
-    Converts the passed MB value to GB value by rounding it down.
+    Converts the passed MB value to GB value by rounding it down with 3 fractional digits in the return value.
+    #>
+    [double] ConvertMBValueToGBValue($mbValue) {
+        return [Math]::Round($mbValue * $this.MegabytesInGB * $this.MegabytesInGB / 1GB, $this.NumberOfFractionalDigits)
```

Comment:
Conversion should be

$mbValue*1MB/1GB

Reply:
Done.